apps/web/app/page.tsx

```diff
@@ -16,12 +18,8 @@ export default function Page(): JSX.Element {
         <Header greeting={greetingData.greet} name={greetingData.name} />
         <ProfilesCarousel profileCardData={profileCardData} />
         <ToolsCarousel toolsCardData={toolsCardData} />
-        {/* <ImageCard
-          imgUrl="/default-pic.png"
-          caption="Desert Collection"
-          tag="Trending"
-        ></ImageCard> */}
         <NavBar navBarData={NavBarData} />
+        <RecentOrderCarousel recentOrderCardData={RecentOrderData} />
```

Comment:
Can you create a new page with /clients/[id] and move this component there? In that way you can also check the Navbar for various pages.